core/src/main/java/org/keycloak/representations/account/UserRepresentation.java

```diff
@@ -106,4 +107,28 @@ public String firstAttribute(String key) {
         return this.attributes == null ? null : this.attributes.containsKey(key) ? this.attributes.get(key).get(0) : null;
     }
 
+    public Map<String, List<String>> toAttributes() {
```

Comment:
We can avoid this method here by providing additional methods to the user profile to pass the user representation in addition to a map of attributes.

Also, the representation for the account is pretty much the same as the one used in the admin rest api. So, we could probably use it in the account too.